### DISCUSSION TITLE
Desktop: black background behind shortcut label

### DISCUSSION THREAD

**Discussion by ghost** [linuxmint/discussions#717] [Ideas]
Currently very light desktop backgrounds will have the same contrast as shortcut labels/text, so it's better to place a little black background behind shortcut labels/text.

![image](https://github.com/user-attachments/assets/79da76d0-64f2-40e6-8e19-b66881acf205)

**Comment by Secret-chest** [comment]
Or add a shadow. This is theme-related. Also, what DE?

**Reply by ghost** [reply]
It's Linux Mint Cinnamon 22. About a shadow effect: not if it's too weak (current case).

I'm using the Windows 8 modern theme from b00merang, but I don't know where to edit desktop labels in the CSS (I've searched for `desktop` there and got no result).

**Reply by Secret-chest** [reply]
I'll try to help when I have time.

**Reply by Jopp-gh** [reply]
I've tried at least a dozen of custom themes, unluckily none of them was able to cover all applications I use (flatpak, deb, appimage) so finally, I gave up to try and kept instead mint-y for everyday use.

To create a theme for Mint is way too complicated - I myself don't have any issues with CSS, which is a very easy to learn code language - but without the right tools, theme-building for Mint is pretty impossible. 

Usually themes invite to customization, at some point you see user interface issues everywhere, so editing a simple label might take you on a long journey. 

Just saying-good night, ehm I meant, good luck with this endeavor.

**Comment by ChooChooAl** [comment]
How did you get that shortcut on the desktop?
If you created a launcher, then all launchers allow for a custom icon.   Opening images.google.com and searching for 'gear icon' will give you a good number of them to pick from.  Save it someplace to hold and use it in the launcher.   I alter all my shortcuts for clarity.

**Reply by ghost** [reply]
But the icon isn't the problem; it's the text (the label) that has a very tiny shadow currently.

**Comment by ChooChooAl** [comment]
Understand.   I have similar issues, I have a slideshow of my wallpaper and I maintain wallpapers that are in the darker range so labels show better.   I even have a black tint that covers the left hand 10% so that even a bit lighter wallpaper will still be dark on that edge where my icons are.    I think you can see it here, the sunflowers are darker on the left.

![Screenshot from 2025-01-09 11-38-02](https://github.com/user-attachments/assets/354dd4c3-72e9-456c-8efd-1a8269d0ac0d)
